homeassistant/helpers/entity_component.py

```diff
@@ -31,7 +30,7 @@
 DEFAULT_SCAN_INTERVAL = timedelta(seconds=15)
 DATA_INSTANCES = "entity_components"
 
-_EntityT = TypeVar("_EntityT", bound=entity.Entity, default=entity.Entity)
+_EntityT = TypeVar("_EntityT", bound=entity.Entity)
```

Comment:
The `Entity` default here is ok, as that's a required type.

Reply:
If you want to revert the whole PR that is ok. I just want to clarify what I don't want added at this point in time.

Reply:
👌🏻 Let me push an update. I'll just revert the `defaults` for `DataUpdateCoordinatorT`, `DataT`, and `_PassiveBluetoothDataUpdateCoordinatorT` then.

Can I leave the TypeVar renaming from `_T` to `_DataT` or should that be reverted as well?

Reply:
The new name is ok with me. 👍